src/test/resources/images

```diff
@@ -0,0 +1 @@
+../../../python/tests/resources/images
```

Comment:
can we rename this file to py-images or something that indicates it's pointing somewhere else? was a bit confused by why smaller and bigger images were in different directories.

Reply:
I moved the images into a sparkdl directory to avoid conflicts, I also renamed the main image directory to "test-image-collection" and the smaller image to "00081101-small-version.png" to make it clear that it's a smaller version of a specific image from the collection.